server/chat-commands/core.ts

```diff
@@ -374,12 +377,17 @@ export const commands: Chat.ChatCommands = {
 
 		return this.parse(message);
 	},
-	msghelp: [`/msg OR /whisper OR /w [username], [message] - Send a private message.`],
+	msghelp: [
+		`/msg OR /whisper [username], [message] - Send a private message.`,
+		`You can also use /whisper, /w, /dm, or /pm. You can also click a username and press Chat.`,
+		`If the other user is offline, use /offlinemsg.`
+				],
 
 	offlinepm: 'offlinemsg',
 	opm: 'offlinemsg',
+	odm: 'offlinemsg',
 	offlinewhisper: 'offlinemsg',
-	ofw: 'offlinemsg',
+	offw: 'offlinemsg',
 	async offlinemsg(target, room, user) {
```

Comment:
I don't think we need these many aliases. Or an ``/offlinepm`` variant at all. Just sending the whisper twice works anyway.

Reply:
I find sending the whisper twice annoying and would like at least one.

Reply:
What if the ``"User is offline. Use ``/offlinepm`` [...]"`` message included a button for (re/offline)-sending?